assets/src/edit-story/units/unitsProvider.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Context from './context';
+
+function UnitsProvider( { width, height, children } ) {
+	const percentToEditorPx = useCallback(
+		( percent, dir ) => dirValue( dir, width, height ) * percent * 0.01,
```

Comment:
I wonder if background images should keep the ratio as well, e.g. use `cover` in the FE and in the editor make the (relatively) shorter side 100%, the other 100%+. UX question?

Reply:
Images should always keep ratio AFAIK.